back-nest/src/authentication/auth.controller.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Body,
+  Controller,
+  Post,
+  Get,
+  Request,
+  HttpCode,
+  HttpStatus,
+  UsePipes,
+  ValidationPipe,
+  UseGuards,
+} from '@nestjs/common';
+
+import { AuthGuard } from './auth.guard';
+import { AuthService } from './auth.service';
+import { SignInDto } from '../dto/signIn.dto';
+
+@Controller('auth')
+export class AuthController {
+  constructor(private authService: AuthService) {}
+
+  @HttpCode(HttpStatus.OK)
+  @Post('login')
+  @UsePipes(new ValidationPipe({ whitelist: true }))
+  signIn(@Body() signInDto: SignInDto) {
```

Comment:
You don't need to include DTO in a name of a function argument
```suggestion
  signIn(@Body() body: SignInDto) {
```